apps/consent/env.ts

```diff
@@ -8,10 +8,12 @@ export const env = createEnv({
   },
   shared: {
     GRAPHQL_ENDPOINT: z.string().default("http://localhost:4002/graphql"),
+    NODE_ENV: z.string().default("development"),
```

Comment:
cf comment on the other repo. 

it's not ideal to have several PR open for review with code overlapping. it adds overhead for the reviewer.